classes/ActionScheduler_WPCLI_Command_Info.php

```diff
@@ -6,25 +6,145 @@ class ActionScheduler_WPCLI_Command_Info extends ActionScheduler_Abstract_WPCLI_
 	 * Execute command.
 	 */
 	public function execute() {
-		$columns = explode( ',', \WP_CLI\Utils\get_flag_value( $this->assoc_args, 'columns', 'status,count' ) );
+		$hooks = \WP_CLI\Utils\get_flag_value( $this->assoc_args, 'hooks', false );
+		$past_due = \WP_CLI\Utils\get_flag_value( $this->assoc_args, 'past-due', false );
+
+		if ( $past_due ) {
+			$this->past_due( $hooks );
+		} else if ( $hooks ) {
+			$this->hooks( $hooks );
+		} else {
+			$this->default();
+		}
+	}
 
+	/**
+	 * Print info on past due actions.
+	 *
+	 * @param string $hooks
+	 */
+	protected function past_due( $hooks ) {
 		$store = ActionScheduler::store();
+		$columns = explode( ',', \WP_CLI\Utils\get_flag_value( $this->assoc_args, 'columns', 'hook,total' ) );
+
+		if ( true === $hooks ) {
+			global $wpdb;
+
+			$query = $wpdb->prepare( "SELECT DISTINCT( post_title ) FROM $wpdb->posts WHERE post_type = %s", ActionScheduler_wpPostStore::POST_TYPE );
```

Comment:
You can't use direct DB queries in WP CLI commands because AS supports custom data stores. This query would only work with the default data store.

Reply:
Thanks, @rrennick! 

https://github.com/Prospress/action-scheduler/pull/279/commits/d418a8681949934614d7d087db7732ba076008a0 introduces `ActionScheduler_wpPostStore::action_hooks()`.